sumpy/e2e.py

```diff
@@ -515,7 +516,9 @@ def get_kernel(self, result_dtype):
     def get_optimized_kernel(self, result_dtype):
         knl = self.get_kernel(result_dtype)
         # FIXME
```

Comment:
```suggestion
```

You're adding the missing transform, after all.

Reply:
Fixed in 2746b9a01e88e749a37308dd0386c6c8ae9ed613